gems/sorbet/test/snapshot/check_one_bazel.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Runs a single srb init test from gems/sorbet/test/snapshot/{partial,total}/*
+
+set -euo pipefail
+shopt -s dotglob
+
+source "gems/sorbet/test/snapshot/logging.sh"
+
+# ----- Option parsing -----
+
+test_name=$1
+
+if [[ "${test_name}" =~ partial/* ]]; then
+  is_partial=1
+else
+  is_partial=
+fi
+
+info "├─ test_name: ${test_name}"
+info "├─ is_partial: ${is_partial:-0}"
+
+# ----- Environment setup -----
+
+repo_root=$PWD
```

Comment:
Idk if this affects your script, but `$PWD` is allowed to have symlinks in it, i.e., it's not the case that

```bash
[ "$PWD" = "$(realpath "$PWD")" ]
```

in all cases (again not sure if this is important for you).

Reply:
`realpath` doesn't exist in the sandbox, so this was the best workaround that I had come up with.